superset/models/sql_lab.py

```diff
@@ -246,6 +246,7 @@ def data(self) -> Dict[str, Any]:
             "database": {"id": self.database_id, "backend": self.database.backend},
             "order_by_choices": order_by_choices,
             "schema": self.schema,
+            "verbose_map": {},
```

Comment:
Present [here](https://github.com/apache/superset/blob/56dcf25cf8ac5ed47235ebacde07200ccba257b3/superset/connectors/base/models.py#L293) for the `superset.connectors.base.models.BaseDatasource` model.